Intacct/ReadMe.md

```diff
@@ -223,31 +288,13 @@ Most organizations will mark the GL Project designation by setting a default Pro
     
     Field Type: Text
 ```
+<div style="page-break-after: always;"></div>
 
 **Important Information about Your Custom Dimension**
-
 - The Name is for internal (Rock) purposes only
 - The Category has to be set to the Intacct Export category in order to be included in the API post
-- The Key is the actual Intacct specific name of the Custom Dimension in all caps, beginning with "GLDIM". Also, you'll notice that the core KFS Dimensions use the format `rocks.kfs.Intacct.CLASSID`. The custom key can be anything you'd like, so long as there is a period before the Dimension name. For example, `org.mychurch.Intacct.GLDIMMYCUSTOMDIM` is a valid Attribute Key.
-- Text attributes are recommended for the Custom Dimensions. However, the value you enter in the attribute on the Financial Account must be the System Info > ID, not the text value. (i.e. Value: MT-Event-P1234, ID: 10005. You must enter 10005 in Rock.)
-
-**Examples:**
-- *Custom Dimension 1:*
-  - Record Name: Restriction
-  - Integration Name: restriction
-  - Rock Key: GLDIMRESTRICTION
-- *Custom Dimension 2:*
-  - Record Name: Custom Project
-  - Integration Name: custom_project
-  - Rock Key: GLDIMCUSTOM_PROJECT
-- *Custom Dimension 2 Value:*
-  - Custom Project: MT - Event - P1234
-  - ID: 10005
-  - Rock Value: 10005
-
-![IntacctCustomDimensionSystemInfoScreenshot](https://user-images.githubusercontent.com/2990519/174348414-42ead26f-0dd5-4c3c-b985-d1fb48141508.jpeg)
-
-
+- The Key is the actual Intacct specific name of the custom Dimension in all caps. Also, you'll notice that the core KFS Dimensions use the format `com.kfs.Intacct.CLASID`. The custom key can be anything you'd like, so long as there is a period before the Dimension name. For example, `org.mychurch.Intacct.MYCUSTOMDIM` is a valid Attribute Key.
+- Since the Dimensions for Accounts are typically plain text, only text attributes are supported for the custom Dimensions
```

Comment:
Please add back these changes that are needed in the documentation if they didn't exist, the readme went into better detail about custom dimensions.
```suggestion
- The Key is the actual Intacct specific name of the Custom Dimension in all caps, beginning with "GLDIM". Also, you'll notice that the core KFS Dimensions use the format `rocks.kfs.Intacct.CLASSID`. The custom key can be anything you'd like, so long as there is a period before the Dimension name. For example, `org.mychurch.Intacct.GLDIMMYCUSTOMDIM` is a valid Attribute Key.
- Text attributes are recommended for the Custom Dimensions. However, the value you enter in the attribute on the Financial Account must be the System Info > ID, not the text value. (i.e. Value: MT-Event-P1234, ID: 10005. You must enter 10005 in Rock.)

**Examples:**
- *Custom Dimension 1:*
  - Record Name: Restriction
  - Integration Name: restriction
  - Rock Key: GLDIMRESTRICTION
- *Custom Dimension 2:*
  - Record Name: Custom Project
  - Integration Name: custom_project
  - Rock Key: GLDIMCUSTOM_PROJECT
- *Custom Dimension 2 Value:*
  - Custom Project: MT - Event - P1234
  - ID: 10005
  - Rock Value: 10005

![IntacctCustomDimensionSystemInfoScreenshot](https://user-images.githubusercontent.com/2990519/174348414-42ead26f-0dd5-4c3c-b985-d1fb48141508.jpeg)
```